runtime/service/src/main/java/org/apache/polaris/service/admin/PolarisServiceImpl.java

```diff
@@ -89,6 +89,7 @@
 
 /** Concrete implementation of the Polaris API services */
 @RequestScoped
+@MainService
```

Comment:
This may not be necessary anymore when using proper interfaces and `@Decorator` / `@Delegate`.

Reply:
You're correct - removed!